NAMESPACE

```diff
@@ -1,6 +1,8 @@
 # Generated by roxygen2: do not edit by hand
 
 export("%>%")
+export(accessible_action_link)
+export(accessible_radio_button)
```

Comment:
These are added by including `@export` in the documentation sections, then running `roxygen2::roxygenise()`.